app/src/test/java/homer/model/lights/LightTest.java

```diff
@@ -0,0 +1,25 @@
+package homer.model.lights;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import homer.DeviceInfoImpl;
+import homer.api.DeviceIdImpl;
+
+/**
+ * Tests Light functionalities.
+ */
+public final class LightTest {
```

Comment:
```suggestion
final class LightTest {
```
test classes should be package private

Reply:
My bad, will change that, too